packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -253,21 +253,18 @@
 				</li>
 				<li class="nav-item">
 					<span class="navbar-text">
-						{if $supportsBulkActions and $selectedItems == $totalItems}
-							{msg desc="Says that all possible items are selected when bulk actions are available"}
-								all-{$totalItems}-items-selected
-							{/msg}
-						{else}
-							{msg desc="Says how many items of the total are selected"}
-								{$selectedItems}-of-{$totalItems}
```

Comment:
I think we should keep this message still, but do not show it when the number of items selected and the total items are the same.

Reply:
Was defined in https://issues.liferay.com/browse/LPS-90796 to use `All Selected` even if the number of selected items is 1.